.default.conf

```diff
@@ -27,16 +27,16 @@ HF_TOKEN=
 MISTRAL_API_KEY=
 # Access token to use when attempting to interact with OpenAI's API
 OPENAI_API_KEY=
-# MLFlow Configuration
-MLFLOW_TRACKING_URI=http://mlflow:5000
-MLFLOW_DATABASE_URL=sqlite:///mlflow.db
-MLFLOW_S3_ROOT_PATH=s3://mlflow
 # S3 Configuration (MinIO)
 MINIO_ROOT_USER=minioadmin
 MINIO_ROOT_PASSWORD=minioadmin # pragma: allowlist secret
 MINIO_API_CORS_ALLOW_ORIGIN="*"
 DEPLOYMENT_TYPE=local
 DATABASE_URL=sqlite:////db-data/local.db
+# MLFlow Configuration
+MLFLOW_TRACKING_URI=http://mlflow:5000
+MLFLOW_DATABASE_URL=sqlite:////db-data/mlflow.db
+MLFLOW_S3_ROOT_PATH=s3://${S3_BUCKET}/mlflow
```

Comment:
Again, use a separate S3 bucket for mlflow if you feel there would be some collision with lumigator itself.